DeepSpeech.ipynb

```diff
@@ -1351,6 +1382,17 @@
     "    \n",
     "    # Loop over the data set for training_epochs epochs\n",
     "    for epoch in range(start_epoch, training_iters):\n",
+    "        \n",
+    "        print\n",
+    "        print\n",
+    "        print\n",
+    "        print \"STARTING Epoch\", '%04d' % (epoch)\n",
```

Comment:
I'd personally rather you just put three '\n's before the string, or one print statement with two '\n's. But preferably the former.